www/include/monitoring/status/Hosts/xml/hostXML.php

```diff
@@ -71,40 +69,42 @@
 /*
  *  Check Arguments from GET
  */
-$o          = $obj->checkArgument("o", $_GET, "h");
-$p          = $obj->checkArgument("p", $_GET, "2");
-$num        = $obj->checkArgument("num", $_GET, 0);
-$limit      = $obj->checkArgument("limit", $_GET, 20);
-$instance   = $obj->checkArgument("instance", $_GET, $obj->defaultPoller);
-$hostgroups = $obj->checkArgument("hostgroups", $_GET, $obj->defaultHostgroups);
-$search     = $obj->checkArgument("search", $_GET, "");
-$order      = $obj->checkArgument("order", $_GET, "ASC");
-$dateFormat = $obj->checkArgument("date_time_format_status", $_GET, "Y/m/d H:i:s");
+$o = $obj->checkArgument("o", $_GET, "h");
+$p = $obj->checkArgument("p", $_GET, "2");
+$num = $obj->checkArgument("num", $_GET, 0);
+$limit = $obj->checkArgument("limit", $_GET, 20);
+$search = $obj->checkArgument("search", $_GET, "");
+$order = isset($_GET['order']) && $_GET['order'] === "ASC"
+    ? "ASC"
+    : "DESC";
 
 $statusHost = $obj->checkArgument("statusHost", $_GET, "");
 $statusFilter = $obj->checkArgument("statusFilter", $_GET, "");
+$criticalityValue = $obj->checkArgument('criticality', $_GET, $obj->defaultCriticality);
```

Comment:
criticality_id is renamed as in fact it's used in the query (line 163) to get a value (varchar) and not an id.